docs/DesignDocs/oe_log_message()_callback_proposal.md

```diff
@@ -15,6 +15,10 @@ function.
 
 The callback API is supposed to solve such problems.
 
+Additionally, oe_set_host_log_level() and oe_set_enclave_log_level() APIs allow
```

Comment:
Ok for this time, but Ideally the design doc changes need to go into a separated PR.